core/src/wallet/bitcoin/BitcoinLikeAccount.cpp

```diff
@@ -553,60 +553,69 @@ namespace ledger {
                 //Store newly broadcasted tx in db
                 //First parse it
                 auto txHash = seq.str();
+                auto optimisticUpdate = Try<int>::from([self, transaction, txHash] () -> int {
```

Comment:
Why just not try{}catch(){}, instead of creating new object, copy self, transaction, txHash and get a mess with call stack during debugging.

Reply:
I understand your concern about the stack, but I will use `Try` class to comply with what we mostly do but I will capture by reference in this context it's ok to do so